spec/ics-003-connection-semantics/README.md

```diff
@@ -334,28 +333,27 @@ function connOpenTry(
     abortTransactionUnless(validateConnectionIdentifier(desiredIdentifier))
     abortTransactionUnless(consensusHeight < getCurrentHeight())
     expectedConsensusState = getConsensusState(consensusHeight)
-    abortTransationUnless(
-      counterpartyChosenConnectionIdentifer === "" ||
```

Comment:
I think `counterpartyChosenConnectionIdentifier` can be removed from the function arguments

Reply:
Aye - removed.